assessments/projects/tikv/self-assessment.md

```diff
@@ -0,0 +1,335 @@
+
+The TiKV project utilizes persistent open groups which each focus on a part of the TiKV project; teams and members provided in the teams folder for each individual project. Each team consists of reviewers, committers, and maintainers:
+* **Reviewers**: Responsible for contributions and code review 
+* **Committees**: Granted write access and usually responsible for certain areas of the project
+  - Each commit message must contain a “Signed-off-by” line for Developer Certificate of Origin
+* **Maintainers**: Responsible for moderating the discussions, managing project release, and proposing new committers or maintainers
+
+In addition, there is a separate infrastructure team, not responsible for the team decision-making process, but that helps on applying the decision. The current list of infrastructure members is provided with their emails and githubs on the TiKV Governance page.
+* **Infrastructure team**: Responsible for setting administration, security, and supporting the existing teams
+
+**Voting**
+* Decisions regarding the project are made by votes on the primary project community repository.
+* Votes are indicated via pull requests under the votes folder.
+* Positive votes require no explanation, while vetoes (negative votes) must have an explanation. 
+* Required vote counts can vary based on the decision but typically require at least 2 positive votes.
+
+* Internal Communication: Slack 
+
+ * Inbound Communication: Twitter, Blog, Reddit, Stack Overflow
+
+* Outbound Communication: None
+
+**Ecosystem**
+* At the time of writing, there are currently four client drivers: Go, Java, Rust, and C
+  - The Go client is the only stable client
+  - The Java and Rust clients are considered unstable
+  - The C client is currently in early development
+
+TiKV provides two separate clients: Raw and Transactional
+* ***Raw***: A lower-level key-value API for interacting directly with individual key-value pairs
+* ***Transactional***: A higher level key value API that provides ACID semantics
+
+## Security issue resolution
+
+***Note:*** When looking for methods of security issue resolution, there was no easily accessible area to discuss each protocol. This could potentially lead to mishaps like major security vulnerabilities being publicly shown as issues
+
+* Bug Fixes or documentation improvement can be implemented via a normal Github Pull Request workload
+* To contribute “substantial changes”, a RFC (request for comments) document provides a consistent and controlled path for new features to enter the project
+  - RFCs must provide summary, motivation, detailed design, drawbacks, alternative, and unanswered questions
```

Comment:
Security issues refers to vulnerabilities in the project. This content is not relevant to the security issue resolution. Please remove them

Reply:
We have updated the section and shortened it according to your feedback. Hopefully, everything is in order now. Thanks!